source/CCustomOpcodeSystem.cpp

```diff
@@ -2285,12 +2302,12 @@ namespace CLEO {
 	//0ADF=2,add_dynamic_GXT_entry %1d% text %2d%
 	OpcodeResult __stdcall opcode_0ADF(CRunningScript *thread)
 	{
-		char *entryName;
-		char text[128];
-		char *str;
-		entryName = readString(thread);
-		str = readString(thread, text, sizeof(text));
-		GetInstance().TextManager.AddFxt(entryName, str);
+		char gxtLabel[8 + 1]; // and terminator character

```

Comment:
isn't the max length of a GXT key 8 (7 characters + null terminator)?

Reply:
Right. I thought opcodes know it is up to 8 without need for terminator. Fixed.